aiida/transports/plugins/ssh.py

```diff
@@ -184,6 +184,16 @@ class SshTransport(Transport):  # pylint: disable=too-many-public-methods
     # to return a suggestion; it must accept only one parameter, being a Computer
     # instance
     _valid_auth_options = _valid_connect_options + [
+        (
+            'use_login_shell', {
+                'default': True,
+                'switch': True,
+                'prompt': 'Use login shell when executing command',
+                'help': ' Avoiding a login shell can help suppress spurious text output'
```

Comment:
```suggestion
                'help': 'Not using a login shell can help suppress any potential spurious text output that can interfere with AiiDA executing commands, '
```
I guess it might not be immediately clear why suppressing spurious output is something desirable, per se.